data/test/vtgate/unsupported_cases.txt

```diff
@@ -201,3 +201,11 @@
 # nextval not supported yet
 "select next value from user"
 "unsupported: NEXT VALUE construct"
+
+# complex expression in parenthesis with limit not supported yet
+"select * from user where (id = 4 AND name ='abc') limit 5"
```

Comment:
Would be nice if we could handle this case also in the future. But we can leave it be for now.